### PR TITLE
Configure pycodestyle

### PR DESCRIPTION
**Pull request by stoivo**
It would be good if we added this as a Unit test, or what do you think?

@randy3k, how would you do it?

### CODE REVIEW

**Comment by randy3k** [comment]
We'll need a sublime dependency of pycodestyle in order write a test case. A simple solution is to run `pycodestyle` on travis.

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ before_install:
       fi
     - git config --global user.email gitsavvy@gitsavvy.com
     - git config --global user.name GitSavvy
+    - pip install --upgrade pip
+    - pip install pycodestyle
```
Comment (randy3k) [inline .travis.yml]: you may want to move the code block https://github.com/divmain/GitSavvy/blob/99c54673fb617b86d869b628d6df064fae78ec23/.travis.yml#L37-L46
to here

Note that, on macOS, `pip` corresponds to python 2, so `pip3` is needed.

Comment (stoivo) [reply .travis.yml]:
```
pip install python-coveralls;
pip install pycodestyle;
```
vs
```
pip install python-coveralls pycodestyle;
```

Comment (randy3k) [reply .travis.yml]: or
```
    - if [ "$TRAVIS_OS_NAME" == "osx" ]; then
            brew update;
            brew install python3;
      fi
    - pip3 install python-coveralls pycodestyle
```

Comment (stoivo) [reply .travis.yml]: How did you check that? 
Am I not used to any CI?

File: .travis.yml [hunk 2]
```diff
@@ -30,16 +30,14 @@ install:
...
 after_success:
     - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
             brew update;
             brew install python3;
-            pip3 install python-coveralls;
-      fi
-    - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
-            pip install python-coveralls;
       fi
+    - pip3 install python-coveralls pycodestyle;
```
Comment (randy3k) [inline .travis.yml]: you need to install `pycodestyle` before running

**Comment by stoivo** [comment]
Travis failed with 
```
$ pycodestyle --statistics --count
/Users/travis/.travis/job_stages: line 57: pycodestyle: command not found
The command "pycodestyle --statistics --count" exited with 
```

@randy3k ?

File: .travis.yml (hunk 1 above)
Comment (randy3k) [reply .travis.yml]: I just tried...

**Comment by randy3k** [comment]
move 
```
    - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
            brew update;
            brew install python3;
      fi
    - pip3 install python-coveralls pycodestyle
```
under `install`

File: .travis.yml (hunk 2 above)
Comment (stoivo) [reply .travis.yml]: move it to install section then ?

Comment (randy3k) [reply .travis.yml]: ya

**Comment by stoivo** [comment]
I will do a fixup rebase

**Comment by randy3k** [comment]
Great. Thanks.

**Pull request merged**